pkg/skaffold/build/gcb/jib.go

```diff
@@ -57,3 +59,22 @@ func (b *Builder) jibBuildSpec(artifact *latest.Artifact, tag string) (cloudbuil
 func fixHome(command string, args []string) []string {
 	return []string{"-c", command + " -Duser.home=$$HOME " + strings.Join(args, " ")}
 }
+
+func jibAddWorkspaceToDependencies(workspace string, dependencies []string) ([]string, error) {
+	dependencyMap := make(map[string]bool)
+	for _, d := range dependencies {
```

Comment:
I wonder if the workspace should be enough? and we don't need to get dependencies like this and add the workspace to it?

Reply:
Yeah, I wasn't sure if it was possible for it to pull in external dependencies from outside the workspace?

Reply:
Good point, I think it's technically possible, but I would find it very strange if someone actually did that.